protocol/0028-GOVE-governance.md

```diff
@@ -263,6 +263,17 @@ Enactment of an asset modification proposal is:
 
 ## 5. Transfers initiated by Governance (post Oregon trail)
 
+## 6. Close market
+
+Any type of market (either fixed expiry market or perpetual) can be closed via a governance vote.
```

Comment:
We should make clear that it's the same voting rules as market update proposals (i.e. LPs vote via ELS, can be overriden by governance token holders).

Reply:
I remember that being mentioned last time we discussed it, but couldn't find anything in the specs about it (probably was looking in a wrong place)